src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/Example1.java

```diff
@@ -0,0 +1,27 @@
+/*
+WhitespaceAround
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespacearound;
+
+// xdoc section -- start
+class Example1 {
+  public Example1(){} // 3 violations
+  public static void main(String[] args) {
+    if (true) { }
+    else{ // 2 violations
+    }
+
+    for (int i = 1; i > 1; i++) {} // 2 violations
+
+    Runnable noop = () ->{}; // 4 violations
```

Comment:
4 violations are not obvious here, please add comment to explain.

Reply:
we still do not have support in test for cases when few violations are on the same line.
I do not recommend to add support for it now, it might be too complicated.
But I agree to change example a bit to keep only 1 violation on line.

May be it is good rule for Examples - one violation per line only because use need to easily catch place of it.
If all agree on this we can add this validation idea to our list of example-checks validation later on.

Reply:
I'd rather put a comment below explaining what the 4 violations are about. Something like this below. Let me know if you agree.
```
// no space after '->' and '{', no space before '{' and '}'
```

Reply:
I am ok to have 4 comments.

But if that is easier, just update example code to have single violation.

Reply:
>I'd rather put a comment below explaining what the 4 violations are about. Something like this below. Let me know if you agree.

Whatever you want to do is fine as long as we point out why we have four violations

Reply:
Done. I also explained the 3 violations below.

Reply:
Done.